projects/klippa/ngx-enhancy-forms/src/lib/form/form.component.ts

```diff
@@ -50,20 +50,38 @@ export class FormComponent {
 	}
 	private disableInactiveFormControl(control: FormControl) {
 		if (this.activeControls.some((e) => e.formControl === control)) {
-			control.enable();
+			if (!control.disabled) {
```

Comment:
I am wondering how much sense this makes now looking at it again.
If it is not disabled (therefore enabled), enabling again does nothing, right?

Reply:
huh, yeah, was that if originally supposed to be:

```
if ( originallyEnabled? ) control.enable();
```

It definitely isn't being called now I and I think that it's not needed becasue the `trySubmit()` handles the re-enabling of states with the `setDisabledStatesForAllControls(originalDisabledStates)`